build.gradle

```diff
@@ -169,7 +169,8 @@ subprojects {
             protobuf_util: "com.google.protobuf:protobuf-java-util:${protobufVersion}",
 
             netty: "io.netty:netty-codec-http2:[${nettyVersion}]",
-            netty_epoll: "io.netty:netty-transport-native-epoll:${nettyVersion}:linux-x86_64",
```

Comment:
Thanks, this is much cleaner and updated my code with this change. In fact, this is the only change i needed.

Reply:
I think you still need to change `netty/shaded/build.gradle`. The `osdetector.classifier` is only used during compilation for grpc-netty-shaded. Since gRPC's build for Linux runs on x86_64, this wouldn't actually change our released binary. What you need is for grpc-netty-shaded to include both the x86_64 and arm64 binaries, which means adding both as a dependency in the build.gradle.

Reply:
In that case, i think my previous changes make more sense. It includes both x86 and arm64 binaries for grpc-netty-shaded. I don't think, `osdetector.classifier` is helpful here given your builds take place on x86 machine. Do you agree?

Reply:
The problem with the previous changes is they were made to places other than netty-shaded for usages that were run immediately. Netty-shaded is the usage that runs on a different platform than it is bult on.

It is fine if this line remained linux-x86_64. And it looks like using osdetector actually breaks on Windows and Mac because there aren't artifacts available for those, so it is probably simpler to leave alone.

Reply:
Understood. I modified code to only add dependency for netty-shaded. I verified that grpc-netty-shaded is including both binaries.